doc/examples/filters/plot_tophat.py

```diff
@@ -0,0 +1,28 @@
+"""
+=======
+Top hat
```

Comment:
Can you write a more explicit title like "removing small bright objects with a top hat"?